release: 11.0.0 (#3906)
rome/tools: website/src/pages/configuration.mdx

This configuration file enables the formatter and sets the preferred indent style and width. The linter is disabled.

## `$schema`

Allows to pass a path to a JSON schema file.

Since version `11.0.0`, a JSON schema file for the `rome.json` is published.

> You can specify a relative path to the schema of the `rome` npm package if `rome` is installed in the `nodue_modules` folder:

Reviewer: ```suggestion
You can specify a relative path to the schema of the `rome` npm package if `rome` is installed in the `node_modules` folder:
```

This configuration file enables the formatter and sets the preferred indent style and width. The linter is disabled.

## `$schema`

Allows to pass a path to a JSON schema file.

Since version `11.0.0`, a JSON schema file for the `rome.json` is published.

> If you installed Rome via a package manager and present inside the
> `node_modules` folder, you can add it like this:

Reviewer 2: ```suggestion
You can specify a relative path to the schema of the `rome` npm package if `rome` is installed in the `nodue_modules` folder:
```

> Default: 1024*1024 (1MB)

 ### `files.ignore`

A list of Unix shell style patterns. Rome tools will ignore files/folders that will

> match these patterns

Reviewer 2: ```suggestion
match these patterns.
```
